protocol/messenger.go

```diff
@@ -1939,7 +1940,17 @@ func (m *Messenger) SendChatMessage(ctx context.Context, message *common.Message
 func (m *Messenger) SendChatMessages(ctx context.Context, messages []*common.Message) (*MessengerResponse, error) {
 	var response MessengerResponse
 
+	generatedAlbumId := uuid.New().String()
 	for _, message := range messages {
+		if message.GetContentType() != protobuf.ChatMessage_IMAGE {
+			generatedAlbumId = ""
+		}
+	}
+
+	for _, message := range messages {
```

Comment:
Can you do only one `for`? You can just check the `ContentType` to decide if you apply the album ID

Reply:
this `for` loop was before me, git marked it as my changes :) but I will do it in previous one.